dotcom-rendering/src/web/components/KeyEventCard.tsx

```diff
@@ -71,7 +71,7 @@ const listItemStyles = (palette: Palette) => css`
 	scroll-snap-align: start;
 
 	${from.desktop} {
-		padding-bottom: ${space[5]}px;
+		padding-bottom: 13px;
```

Comment:
If you use `${space[3]}px;` would it look good enough? Cause I think it's usually recommended to use these spaces if possible. And I know spece[3] would result is 12 but I think maybe 1 px wouldn't be much. @oliverlloyd do you have any advice on this?

Reply:
Sure, I don't think it would be an issue if we used `${space[3]}px;`.

Reply:
+1 to this comment; definitely use source here if we can!

Reply:
Thanks @marjisound and @abeddow91, this has now been updated.